docs/k0sctl-install.md

```diff
@@ -1,54 +1,74 @@
-# Installing with k0sctl
+# Install using k0sctl
```

Comment:
Note that when updating page titles, you should also update the navigation accordingly. See the file mkdocs.yml in the parent directory.

Reply:
Thinking that I should update the mkdocs.yml only once a title change has been merged.

Reply:
both changes has to be done simultaneously, otherwise the docs rendering gets "broken"

Reply:
OK. Understood.